### PR TITLE
Cleaned up the code; Clearer instructions; Builds now.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by sevistatic — COMMENTED** [review]
Unfortunately, these diffs are ugly and hard to parse, but what exactly changed?

It looks like it's mostly removing some multi-line comments and changing where my open braces begin. It also looks like some files were removed, such as AskButton.java. Why?

**Comment by ghost** [comment]
My changes:
* Fixed build errors in the project so it compiles and runs
* Removed files that only have methods that exist elsewhere.
* Removed comments that were out of date or otherwise didn't cohere with their associated methods.
* Renamed the doStuff() methods to pushButton()
* Improved the readability of user output
* Changed the "displayed names" of the characters to more ridiculous ones. I didn't change the internal character identifiers referenced in feature descriptions though



Adam Bryant, Ph.D.
Assistant Professor of Computer Science & Engineering
Wright State University
(937) 503-8757, FAX (937) 775-5133
http://people.wright.edu/adam.bryant/

**Comment by sevistatic** [comment]
still figuring out how to do all this, but i had some merge conflicts. i pulled down your pull request, made the changes and updated my master branch with them. i am not sure how to have done that in a way that uses your pull request, so i'mma close it for now, but i did roll in your changes.

also, it turns out the logic for asking questions is broken somehow, so i guess that's next
